packages/babel-plugin-ember-test-metadata/get-normalized-file-path.js

```diff
@@ -11,13 +11,17 @@ const {
  * @param {Object} opts Babel state.file.opts which include root and filename props
  * @param {string} opts.packageName the name of the package as specified in Babel plugin options
  * @param {boolean} opts.isUsingEmbroider whether building using Embroider as specified in Babel plugin options
+ * @param {boolean} opts.getCustomNormalizedFilePath custom function to get normalized file path as specified in Babel plugin options
  * @param {boolean} opts.projectRoot custom relative path to the project's root as specified in Babel plugin options
  * @param {string} opts.filename the absolute perceived path of the file being visited
  * @param {string} opts.root the absolute root project path as seen on disk
  * @returns {string} E.g. tests/acceptance/my-test.js
  */
-function getNormalizedFilePath({ packageName, isUsingEmbroider, projectRoot, filename, root }) {
-  if (!isUsingEmbroider) {
+function getNormalizedFilePath({ packageName, getCustomNormalizedFilePath, isUsingEmbroider, projectRoot, filename, root }) {
+  if (typeof getCustomNormalizedFilePath === 'function') {
+
+    return getCustomNormalizedFilePath(filename, packageName, projectRoot);
```

Comment:
I'd err on the side of caution and just pass through all the parameters, as you never know what other use cases we'll discover in the near future.